suzieq/engines/pandas/interfaces.py

```diff
@@ -220,6 +220,8 @@ def summarize(self, **kwargs) -> pd.DataFrame:
             ('ifDownCnt', 'state != "up" and adminState == "up"', 'ifname'),
             ('ifAdminDownCnt', 'adminState != "up"', 'ifname'),
             ('ifWithMultipleIPCnt', 'ipAddressList.str.len() > 1', 'ifname'),
+            ('errDisabledCnt', 'state == "errDisabled" & adminState == "up"',
```

Comment:
Just a minor change, the second string defines a pandas query string (you can read about it [here](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.query.html)). 

As it is in python, `&` is the bitwise `AND` operation, `and` is the boolean `AND` operation instead.

```suggestion
            ('errDisabledCnt', 'state == "errDisabled" and adminState == "up"',
```